src/components/App/App.tsx

```diff
@@ -109,7 +110,8 @@ const IndexWithProviders = ({ climbingAreas }: IndexWithProvidersProps) => {
       <Loading />
       {!directions && <SearchBox />}
       {directions && <DirectionsBox />}
-      {featureShown && !isMobileMode && <FeaturePanelOnSide />}
+      {featureShown && !isMobileMode && isMounted && <FeaturePanelOnSide />}
```

Comment:
FYI - this isMounted breaks the server-side-rendering for featurepanel (https://github.com/zbycz/osmapp/issues/623)

I reverted it here:
- https://github.com/zbycz/osmapp/pull/648

I understand you wanted to remove the Panel while loading on the mobile, but this is not a good way to do it 🙂  some CSS media queries is the way.

Reply:
I've referenced this in a new issue #690